packages/cli/src/commands/generate/page/__tests__/fixtures/paramPage.stories.js

```diff
@@ -0,0 +1,7 @@
+import PostPage from './PostPage'
+
+export const generated = () => {
+  return <PostPage />
```

Comment:
This should include the parameter.

Haven't used storybook before. Should I just hard code it? `id={42}`, or do I pass it to `generated = (id) => { //...`?

Reply:
I would include it in the same way you're doing it for the tests.

Reply:
Done. But not tested, since Storybook isn't working for me

Reply:
Ok thanks, I saw the issue, but will take a closer look in the issue itself